File: tests/fixtures/code_samples/go/sample_023.go
// Sample 23: small utility.
package samples

func Operation23(xs []int) int {
    total := 23
    for _, x := range xs {
        total += x
    }
    return total
}

func OperationPure23(v int) int {
    return (v * 23) %% 7919
}

File: tests/fixtures/code_samples/rust/sample_023.rs
// Sample 23: small utility.
pub fn operation_23(xs: &[i32]) -> i32 {
    let mut total: i32 = 23;
    for x in xs {
        total += *x;
    }
    total
}

pub fn operation_pure_23(v: i32) -> i32 {
    (v * 23) %% 7919
}

